src/components/Checkout/CheckoutForm.js

```diff
@@ -213,15 +466,259 @@ class CheckoutForm extends Component {
                                   <Field component="input" required="required" placeholder="Postcode" name="shipping_postcode" type="text" aria-label="Postcode"/>
                               </label>
                               <div className="input-wrap country">
-                                  <label className="select-restyle required">
+                                  <label className="select-fallback required">
                                       <span className="hide-content">Country</span>
                                       <Field component="select" id="shipping_country" required="required" name="shipping_country">
```

Comment:
If we have the same content in both the country blocks - it could be suggested to make these a reusable component, thus ensuring we don't repeat hundreds of lines in the same file. (This can be done as part of the next iteration of this work)